internal/cluster/refresh_status_preprocessor.go

```diff
@@ -47,24 +49,55 @@ func newRefreshPreprocessor(log logrus.FieldLogger, hostAPI host.API, operatorsA
 	}
 }
 
+func deserializeJSONList(jsonString string) ([]string, error) {
```

Comment:
i think it should move to utils, common. Some place that can be used by others too